### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ChrisTofuLee**
updated title and added semantic tags on first update then consolidated multiple classes to single ones and cleaned up css file to follow new formats

### CODE REVIEW

File: Homework/Develop/index.html [hunk 1]
```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
                     <a href="#social-media-marketing">Social Media Marketing</a>
                 </li>
             </ul>
-        </div>
+
+        </nav>
+    </header>
+
+    <div class="hero">
```
Comment (surajverma2587) [inline Homework/Develop/index.html]: Research the attribute called `title` and see if you can add a `title` attribute to the `div`. Also, think of why a `title` attribute could be useful for this `div`.

[title](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)

**HINT:** tooltip

File: Homework/Develop/index.html [hunk 2]
```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
...
+
     </div>
-    <div class="hero"></div>
-    <div class="content">
-        <div class="search-engine-optimization">
-            <img src="./assets/images/search-engine-optimization.jpg" class="float-left" />
+    <Main>
```
Comment (surajverma2587) [inline Homework/Develop/index.html]: Are you sure it is `Main`. Remember that all HTML tags are lowercase. Please correct.

File: Homework/Develop/index.html [hunk 3]
```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
...
+        <div id="search-engine-optimization" class="service-section">
```
Comment (surajverma2587) [inline Homework/Develop/index.html]: The `main` tag has 3 `divs` and could we replace these `divs` with a semantic HTML element? It seems like the 3 `divs` are 3 main sections of the main content of the webpage. Please use a semantic HTML element here.

File: Homework/Develop/index.html [hunk 4]
```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
...
                 Social media continues to have a sizable influence on buying habits. Social media marketing helps you determine which platforms are suited to your brand, using analytics to find the right markets and increase your lead generation.
             </p>
         </div>
     </div>
-    <div class="benefits">
-        <div class="benefit-lead">
+    </Main>
+
+    <aside>
+        <div class="benefit">
```
Comment (surajverma2587) [inline Homework/Develop/index.html]: Thing for you to research on: Can we use `section` tags within an `aside` tag? Are there other ways in which we can list out the 3 benefits within the `aside` tag? If you change anything please check the HTML page to see if there are any breaking changes.

File: Homework/Develop/assets/css/style.css [hunk 5]
```diff
@@ -8,39 +8,43 @@ body {
...
     font-size: 48px;
 }
 
-.header h1 .seo {
+header h1 .seo {
     color: #d9dcd6;
 }
 
-.header div {
+Nav {
```
Comment (surajverma2587) [inline Homework/Develop/assets/css/style.css]: Please check this to make sure you have the right case

File: Homework/Develop/assets/css/style.css [hunk 6]
```diff
@@ -8,39 +8,43 @@ body {
     background-color: #d9dcd6;
```
Comment (surajverma2587) [inline Homework/Develop/assets/css/style.css:9]: Maybe worth investigating if we can add a `font-family` here if it being used across the web page and for cases when they are different we can overwrite them in the respective styles.

File: Homework/Develop/assets/css/style.css [hunk 7]
```diff
@@ -8,39 +8,43 @@ body {
...
 }
 
-.header div {
+Nav {
     padding-top: 15px;
     margin-right: 20px;
     float: right;
     font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
     font-size: 20px;
 }
```
Comment (surajverma2587) [inline Homework/Develop/assets/css/style.css:39]: Maybe worth adding some comments for sections of your style sheet in improve readability.

**NOTE:** `/*  This is a comment in CSS */`

**Review by surajverma2587 — CHANGES_REQUESTED** [review]
Please address the comments in this PR

File: Homework/Develop/index.html (hunk 2 above)
Comment (ChrisTofuLee) [reply Homework/Develop/index.html]: thanks, made the changes in here 8b32038a16205c6727065cc1712ce5cd6c7a0285

File: Homework/Develop/index.html (hunk 3 above)
Comment (ChrisTofuLee) [reply Homework/Develop/index.html]: didnt know this, thanks, sorted this out here 8b32038a16205c6727065cc1712ce5cd6c7a0285

File: Homework/Develop/assets/css/style.css (hunk 5 above)
Comment (ChrisTofuLee) [reply Homework/Develop/assets/css/style.css]: updated this here thanks 8b32038a16205c6727065cc1712ce5cd6c7a0285

File: Homework/Develop/index.html (hunk 4 above)
Comment (ChrisTofuLee) [reply Homework/Develop/index.html]: Changed the 'divs' to 'ul' 'li' to identify them better. updated the CSS file to reflect these changes here 84fbd8554660714265b24b0b6ae06cc5bd5d3a16

File: Homework/Develop/assets/css/style.css (hunk 6 above)
Comment (ChrisTofuLee) [reply Homework/Develop/assets/css/style.css:9]: updated the body tag to include the font family and removed the repeated ones in other tags here dd90339a4aa6736e8ec16f222bd65186e4b010a9

File: Homework/Develop/assets/css/style.css (hunk 7 above)
Comment (ChrisTofuLee) [reply Homework/Develop/assets/css/style.css:39]: added the comments within the CSS file here dd90339a4aa6736e8ec16f222bd65186e4b010a9

File: Homework/Develop/index.html (hunk 1 above)
Comment (ChrisTofuLee) [reply Homework/Develop/index.html]: thanks, looked into this and added the title with an appropriate alt tag here 0ea10bfbe5b3ba05d793c8a6115da7b3169938f4

**Review by surajverma2587 — APPROVED** [review]
Look really good and I am happy for you to merge it.

**Pull request merged**